src/embed/ts-embed.ts

```diff
@@ -507,6 +507,28 @@ export class TsEmbed {
         }
     };
 
+    /**
+     * Refresh the auth token if the autoLogin is true and the authType is TrustedAuthTokenCookieless
+     * @param _
+     * @param responder
+     */
+    private tokenRefresh = async (_: any, responder: any) => {
+        const { authType, autoLogin } = this.embedConfig;
+        const isAutoLoginTrue = autoLogin ?? (authType === AuthType.TrustedAuthTokenCookieless);
```

Comment:
We have the same code for AuthExpire embed event. Can you please check and remove the duplicate code.

Reply:
yeah also we dont need to check for autologin and cookieless right ? if embed is requesting a token we should just send it ?